### PR TITLE
Add Server-ManageLeaderstats.lua

### PR DESCRIPTION
**Pull request by WalkerOfBacon**
First PR on anything btw

### CODE REVIEW

**Review by NoobBucket — CHANGES_REQUESTED** [review]
You will need to change/removed "cs" and "rs" from both commands table as they are both used by other existing commands.

`Commands = {"createstat", "cstat", "cs"};`
`Commands = {"removestat", "rstat", "rs"};`

**Comment by WalkerOfBacon** [comment]
> You will need to change/removed "cs" and "rs" from both commands table as they are both used by other existing commands.
> 
> `Commands = {"createstat", "cstat", "cs"};` `Commands = {"removestat", "rstat", "rs"};`

got it, thanks

File: Server/Server-Leaderstats.lua [hunk 1]
```diff
@@ -77,4 +77,22 @@ return function(Vargs)
...
+		Prefix = Settings.Prefix;
+		Commands = {"removestats", "rstats"};
+		Description = "Removes all the stats";
+		AdminLevel = "Moderators";
+		Function = function(plr)
+			local thestats = plr.leaderstats:GetChildren()
+			
+			if thestats then
+				plr.leaderstats:ClearAllChildren()
+				wait(1)
```
Comment (watameln) [inline Server/Server-Leaderstats.lua:91]: Wait is deprecated
https://create.roblox.com/docs/reference/engine/libraries/task

Comment (WalkerOfBacon) [reply Server/Server-Leaderstats.lua:91]: I tested it and it worked.
Ill check again later

Comment (WalkerOfBacon) [reply Server/Server-Leaderstats.lua:91]: Just checked and its not deprecated

Comment (LiveFireExercise) [reply Server/Server-Leaderstats.lua:91]: `task.wait` is more accurate

Comment (WalkerOfBacon) [reply Server/Server-Leaderstats.lua:91]: alr

**Pull request closed without merging**